readme.md

```diff
@@ -212,7 +213,7 @@ The `PreferencesWindowController` adheres to the [Apple HIG](https://developer.a
 - [DockProgress](https://github.com/sindresorhus/DockProgress) - Show progress in your app's Dock icon
 - [More…](https://github.com/search?q=user%3Asindresorhus+language%3Aswift)
 
-You might also like my [apps](https://sindresorhus.com/apps).
+You might also like Sindre's [apps](https://sindresorhus.com/apps).
```

Comment:
You should add a link to your apps too.

Something like:

> You might also like [Sindre's]() and [Christian's]() apps.

Reply:
Will do later with another PR for a list of my apps.